src/components/bookform.jsx

```diff
@@ -29,13 +29,14 @@ function BookForm() {
   };
 
   return (
-    <div>
-      <h2>Add New Book</h2>
+    <div className="form-div">
+      <hr />
+      <h2>ADD NEW BOOK</h2>
       <form onSubmit={handleSubmit}>
```

Comment:
- [ ] Please note that the font-family for the _**"ADD NEW BOOK"**_ text is `Montserrat`. Kindly include it in this style rule for this element. 👍 🖱️